app/bundles/CoreBundle/Views/Default/slim.html.twig

```diff
@@ -1,13 +1,18 @@
 <!DOCTYPE html>
 <html>
-    {% include('MauticCoreBundle:Default:head.html.twig') %}
+    
+    {{ include('MauticCoreBundle:Default:head.html.twig') }}
     <body>
         {{ outputScripts('bodyOpen') }}
+       
         <section id="app-content" class="container content-only">
             {{ include('MauticCoreBundle:Notification:flashes.html.twig', {
                 alertType: 'standard'
             }) }}
-            {{ include('MauticCoreBundle:Default:output.html.twig') }}
+
+            {# {{ include('MauticCoreBundle:Default:output.html.twig') }} #}
+            {% block _content %}{% endblock %}
+
```

Comment:
Looking at the content of `output.html.twig` it looks like any template that extends `slim.html.twig` would now need to use `_content` instead of `content`.

Reply:
I think this is what we want, when looking at the php template? Otherwise you can't print the content of the templates that use slim and base.

Reply:
I see what you're saying.